flaml/tune/tune.py

```diff
@@ -12,11 +12,13 @@
 
     assert ray_version >= "1.0.0"
     from ray.tune.analysis import ExperimentAnalysis as EA
+    from ray.tune.schedulers import TrialScheduler

```

Comment:
Where is TrialScheduler used?

Reply:
It was used in the type of the `scheduler` argument. But I removed the type. Removed the import as well now.